setup/docker.rst

```diff
@@ -0,0 +1,56 @@
+.. index:: Docker
+
+Using Docker with Symfony
+=========================
+
+Can you use Docker with Symfony? Of course! And several tools exist to help,
+depending on your needs.
+
+Complete Docker Environment
+---------------------------
+
+If you'd like a complete Docker environment (i.e. where PHP, web server, database,
+etc. are all in Docker), check out `https://github.com/dunglas/symfony-docker`_.
+
+Alternatively, you can install PHP on your local machine and use the
+:ref:`symfony binary Docker integration <symfony-server-docker>`. In both cases,
```

Comment:
Link to https://symfony.com/doc/current/setup/symfony_server.html#docker-integration instead?

Reply:
I double-checked - that IS where this link goes :). Unless you want me to link to its headline directly (not using `ref` somehow?

Reply:
Oh, sorry. I misread the reference you added in `setup`. I would remove that one and directly link to `setup/symfony_server`.

I now see it's the reference used on line 52 in this file.